### PR TITLE
Updated Proto Files | Fixed FunctionsRequest

### PR DESCRIPTION
**Pull request by AliabbasMerchant**
@YourTechBud

### CODE REVIEW

**Comment by AliabbasMerchant** [comment]
@YourTechBud User Man also done

File: build.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apply plugin: 'java'
-//apply plugin: 'com.google.protobuf'
+apply plugin: 'com.google.protobuf'
 
-group 'com.spaceuptech.api.core'
+group 'com.spaceuptech.space_api'
```
Comment (YourTechBud) [inline build.gradle:4]: Why did we do this change?

File: src/main/java/com/spaceuptech/space_api/sql/SQL.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        Meta meta = Transport.makeMeta(config.projectId, null, db, config.token);
+        Transport.profile(config.stub, id, meta, listener);
+    }
+
+    public void profiles(Utils.ResponseListener listener) {
+        Meta meta = Transport.makeMeta(config.projectId, null, db, config.token);
+        Transport.profiles(config.stub, meta, listener);
+    }
+
+    public void editProfile(String id, String email, String name, String password, Utils.ResponseListener listener) {
```
Comment (YourTechBud) [inline src/main/java/com/spaceuptech/space_api/sql/SQL.java]: `name`, `email` and `password` should be taken as an object.

File: src/main/java/com/spaceuptech/space_api/sql/SQL.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+    // CREATE
+    public Insert insert(String collection) {
+        return new Insert(this.db, this.config, collection);
+    }
+
+
+    // READ
+    public Get get(String collection) {
+        return new Get(this.db, this.config, collection, "all");
```
Comment (YourTechBud) [inline src/main/java/com/spaceuptech/space_api/sql/SQL.java]: can we make `all`, `one`, `upsert`, etc. constants?

File: src/main/java/com/spaceuptech/space_api/sql/SQL.java (hunk 2 above)
Comment (YourTechBud) [reply src/main/java/com/spaceuptech/space_api/sql/SQL.java]: Or maybe we should make a builder instead

**Review by YourTechBud — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 1 above)
Comment (AliabbasMerchant) [reply build.gradle:4]: That was already updated in the proto file, so I had to do the change.
Also, everywhere we have named the API as `space_api`

File: src/main/java/com/spaceuptech/space_api/sql/SQL.java (hunk 3 above)
Comment (AliabbasMerchant) [reply src/main/java/com/spaceuptech/space_api/sql/SQL.java]: Cool

File: build.gradle (hunk 1 above)
Comment (YourTechBud) [reply build.gradle:4]: Got it

File: src/main/java/com/spaceuptech/space_api/sql/SQL.java (hunk 2 above)
Comment (AliabbasMerchant) [reply src/main/java/com/spaceuptech/space_api/sql/SQL.java]: Added a Class

**Comment by AliabbasMerchant** [comment]
@YourTechBud Done

**Pull request merged**